src/graphql-types/ReasonApolloQuery.re

```diff
@@ -8,27 +8,46 @@ module Get = (Config: ReasonApolloTypes.Config) => {
     | Loading
     | Error(apolloError)
     | Data(Config.t);
+  [@bs.deriving abstract]
+  type updateQueryOptions = {
+    [@bs.optional]
+    fetchMoreResult: Config.t,
```

Comment:
What about fetchMoreResult? It should be a Js.Json.t no?

Reply:
Yes, I believe so.